modules/table/src/integrationTest/java/org/apache/ignite/distributed/ReplicasSafeTimePropagationTest.java

```diff
@@ -86,6 +83,8 @@ public class ReplicasSafeTimePropagationTest extends IgniteAbstractTest {
     @InjectConfiguration("mock: { fsync: false }")
     private RaftConfiguration raftConfiguration;
 
+    private static final int MAX_CLOCK_SKEW = 500;
```

Comment:
max clock skew is a part of configuration, why not to inject it? Or at least adding a todo that will let us not to forget to inject it when maxClockSkew will be extracted from SchemaSynchronizationConfiguration.

Reply:
This test was already written this way. RaftGroupOptions is created manually. Let's keep it as is for simplicity.
Adding TODO requires a ticket - IMO overkill for this case.

Reply:
🆗